config/versions/v5.go

```diff
@@ -0,0 +1,73 @@
+package versions
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/buger/jsonparser"
+)
+
+// Version5 implements ConfigVersion
+type Version5 struct {
```

Comment:
```suggestion
type Version5 struct {}
```

This will trigger gofumpt for when you rebase